index.html

```diff
@@ -1,10 +1,73 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="ru">
 <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" />
+    <meta name="description" content="homework" />
+    <meta name="keywords" content="lists tables and forms" />
+    <meta name="author" content="Alexey Tretyakov" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles/style.css">
     <title>lesson</title>
 </head>
 <body>
-<h1>lesson 1</h1>
+<div class="advantages">
+    <h2 class="heading">What do we offer</h2>
+    <hr class="highlight">
+<!--Var 2-->
```

Comment:
this is a really bad practice in your comment. it will be impossible to adaptive for different devices without crunches
it's better to create an html structure according to components
![image](https://user-images.githubusercontent.com/25488411/156513688-56bdd416-10ac-479f-84f2-8efd253f182b.png)